ansible/tasks/load_vault_credentials.yml

```diff
@@ -27,3 +27,5 @@
   loop_control:
     loop_var: secret
   no_log: true
+  vars:
```

Comment:
VENV containing vault module not needed to execute this ansible command.
We should remove this var from the execution of this task